src/components/CityCoinMiningStats.js

```diff
@@ -3,40 +3,48 @@ import { getMiningStatsAtBlock } from '../lib/citycoin';
 
 export function CityCoinMiningStats(blockHeight) {
   const [miningStats, setMiningStats] = useState();
+  const [totalMiners, setTotalMiners] = useState();
+  const [totalAmountStx, setTotalAmountStx] = useState();
+  const [totalAmountToCity, setTotalAmountToCity] = useState();
+  const [totalAmountToStackers, setTotalAmountToStackers] = useState();
 
   useEffect(() => {
     getMiningStatsAtBlock(blockHeight.value).then(result => {
       setMiningStats(result);
+      setTotalMiners(result.value.value.minersCount.value);
+      setTotalAmountStx(result.value.value.amount.value / 1000000);
+      setTotalAmountToCity(result.value.value.amountToCity.value / 1000000);
+      setTotalAmountToStackers(result.value.value.amountToStackers.value / 1000000);
     });
-  }, []);
+  }, [setMiningStats]);
 
   return (
     <>
       {miningStats ? (
         <>
-          <h3>Mining Stats (Block {blockHeight.value})</h3>
-          <div className="container mb-3">
-            <div className="row">
-              <div className="col-6">
-                <div className="row">
-                  <div className="col-6">Amount</div>
-                  <div className="col-6">Data</div>
-                </div>
-                <div className="row">
-                  <div className="col-6">Total Miners</div>
-                  <div className="col-6">Data</div>
-                </div>
-              </div>
-              <div className="col-6">
-                <div className="row">
-                  <div className="col-6">Amount to City</div>
-                  <div className="col-6">Data</div>
-                </div>
-                <div className="row">
-                  <div className="col-6">Amount to Stackers</div>
-                  <div className="col-6">Data</div>
-                </div>
-              </div>
+          <div className="row">
+            <div className="col-lg-6">Miners</div>
+            <div className="col-lg-6">{totalMiners ? totalMiners : null}</div>
```

Comment:
Some indication what is happening, like loading or failed?

Reply:
Good point - I've only seen errors when running locally (usually related to CORS), but added the text `Please Refresh` in 3d4be76